projects/jsf/src/lib/form-content/form-controls/form-control-base.ts

```diff
@@ -39,16 +39,21 @@ export class FormControlBase extends ContentBaseComponent implements OnInit {
       ${errors.invalidEmails.join(', ')}`;
     } else if (hasRequiredPattern && errors.pattern.requiredPattern.toString() === URI_REGEX.toString()) {
       return 'Please enter a valid url.';
+    } else if (hasRequiredPattern) {
+      return `Please enter a valid value.`;
```

Comment:
Have you guys discussed passing a custom error message along in the schema to display it here?